scripts/dev/e2e.py

```diff
@@ -17,26 +17,22 @@
 import sys
 import yaml
 
-TEST_RUNNER_NAME = "test-runner"
+TEST_POD_NAME = "operator-sdk-test"
 
 
-def _load_testrunner_service_account() -> Dict:
-    return load_yaml_from_file("deploy/testrunner/service_account.yaml")
+def _load_test_service_account() -> Dict:
+    return load_yaml_from_file("deploy/operator/service_account.yaml")
 
 
-def _load_testrunner_role() -> Dict:
-    return load_yaml_from_file("deploy/testrunner/role.yaml")
+def _load_test_role() -> Dict:
+    return load_yaml_from_file("deploy/operator/role.yaml")
 
 
-def _load_testrunner_role_binding() -> Dict:
-    return load_yaml_from_file("deploy/testrunner/role_binding.yaml")
+def _load_test_role_binding() -> Dict:
+    return load_yaml_from_file("deploy/operator/role_binding.yaml")
 
 
-def _load_testrunner_cluster_role_binding() -> Dict:
```

Comment:
Do we no longer need to load the cluster role binding here? I see that the old changes loaded the cluster role binding but not the cluster role- is that why?